kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/iptables/PortForwardRule.java

```diff
@@ -128,10 +130,27 @@ public PortForwardRule permittedMAC(String permittedMAC) {
     }
 
     public PortForwardRule sourcePortRange(String sourcePortRange) {
-        this.sourcePortRange = sourcePortRange;
+        parsePortRange(sourcePortRange);
         return this;
     }
 
+    private void parsePortRange(String sourcePortRange) {
```

Comment:
Do we need some check on this field? For example that startPort <= endPort?

Reply:
We could. But we don't have a way to report the exception. We could act safely by fixing something. The question is, what?

Reply:
Let's see it from another pov: what happens if we set a wrong range in the current implementation? Is there an exception when the rules is applied?

Reply:
I verified that in the form in the web console there is a check for this...

Reply:
True. But the code does not protect itself. For example in the case someone updates the configuration outside of the Web UI.
I think to do so we will need some breaking changes. But probably this is outside the scope of this PR.

Reply:
@MMaiero I totally agree. So, leave it as is and let's open an issue if needed.